providers/src/airflow/providers/apache/kafka/hooks/consume.py

```diff
@@ -18,9 +18,23 @@
 
 from typing import Sequence
 
-from confluent_kafka import Consumer
+from confluent_kafka import Consumer, KafkaError
 
 from airflow.providers.apache.kafka.hooks.base import KafkaBaseHook
+from airflow.utils.module_loading import import_string
+
+
+class KafkaAuthenticationError(Exception):
```

Comment:
One NIT - could you please also add some description about that to the documentation https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-apache-kafka/hooks.rst

Reply:
I added documentation for `KafkaAuthenticationError`